proposed/2021/TargetFrameworkAndPlatformsFilters.md

```diff
@@ -0,0 +1,196 @@
+
+## Rationale and alternatives
+
+<!-- Why is this the best design compared to other designs? -->
+<!-- What other designs have been considered and why weren't they chosen? -->
+<!-- What is the impact of not doing this? -->
+The .NET ecosystem currently is divided into two major cohorts: developers using .NET Framework and developers using .NET Core & later(this includes .NET 5+). As newer versions of .NET come out, it is especially important that new & existing developers can filter for what they are looking for while browsing for packages. Whether it's the first job out of college for a .NET developer maintaining a legacy app that targets .NET Framework, or if it's a budding startup wanting to target against the latest .NET SDK, they should be able to easily filter for their job-to-be-done.
+
+### Jobs To Be Done
+
+- James is a desktop developer interested in finding database ORM packages that support the three major desktop OSes: Windows, macOS, and Linux to bring to an architecture review meeting for a new system he is building. He would like to filter database packages that support .NET Core and run on these three platforms.
+- Jules is a game developer using the cloud for their game's multiplayer leaderboard services. They would like to filter game backend platform packages that support the latest .NET or include .NET Standard support to write cross-platform code with and can run on Windows, Android, and iOS for the game they are developing.
+- Katherine is a fresh college graduate who is supporting an old .NET application at her first job. She would like to filter packages that support .NET Framework 4.8 and run on Windows for a new logging feature she is working on this sprint.
+
+## Prior Art
+
+<!-- What prior art, both good and bad are related to this proposal? -->
+<!-- Do other features exist in other ecosystems and what experience have their community had? -->
+<!-- What lessons from other communities can we learn from? -->
+<!-- Are there any resources that are relevant to this proposal? -->
+
+- [pub.dev](https://pub.dev/) has similar filters on the search page for dart & flutter.
+- [pypi](https://pypi.org/) has an extensive filter list of frameworks, OS, and programming languages.
+- [debian](https://packages.debian.org/) has filters for release suites and supported architectures.
+- [winget](https://docs.microsoft.com/windows/package-manager/winget/search) allows you to filter by a supported app moniker.
+
+## Unresolved Questions
+
+<!-- What parts of the proposal do you expect to resolve before this gets accepted? -->
+<!-- What parts of the proposal need to be resolved before the proposal is stabilized? -->
+<!-- What related issues would you consider out of scope for this proposal but can be addressed in the future? -->
+
+- Should there be an advanced option to disable .NET Standard from being implicit/on-by-default for more exact matches?
```

Comment:
Why? What benefit to customers would there be to exclude .NET Standard-only libraries from search results?

Reply:
The original thinking was to be able to search for platform specific assets rather than .NET Standard as a fallback.

Reply:
Added as an unresolved question for now.